ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -207,6 +208,8 @@ open class DrawerController: UIViewController {
     }
     @objc open var presentationBackground: DrawerPresentationBackground = .black
 
+    @objc open weak var passThroughView: UIView?
```

Comment:
there needs to be documentation about this variable

Reply:
Fixed!